hack/cmds/commands.json

```diff
@@ -1577,6 +1578,65 @@
     "since": "1.0.0",
     "group": "generic"
   },
+      "DELIFEQ": {
+          "summary": "Delete key if value matches string.",
+          "complexity": "O(1)",
+          "group": "string",
+          "since": "9.0.0",
+          "arity": 3,
+          "function": "delifeqCommand",
+          "command_flags": [
+              "WRITE"
+          ],
+          "acl_categories": [
+              "STRING"
+          ],
+          "key_specs": [
+              {
+                  "flags": [
+                      "RM",
+                      "ACCESS",
+                      "DELETE"
+                  ],
+                  "begin_search": {
+                      "index": {
+                          "pos": 1
+                      }
+                  },
+                  "find_keys": {
+                      "range": {
+                          "lastkey": 0,
+                          "step": 1,
+                          "limit": 0
+                      }
+                  }
+              }
+          ],
+          "reply_schema": {
```

Comment:
We don’t need the command_flags, acl_categories, key_specs, and reply_schema. Could you help remove them? Also please make the indentation consistent.